src/strategies/otterspace-badges/index.ts

```diff
@@ -52,7 +52,8 @@ function getBadgeWeight(specs: any[], badgeSpecID: string): number {
   if (specs && specs.length > 0) {
     const specConfig = specs.find((spec: any) => spec.id === badgeSpecID);
     badgeWeight =
-      specConfig && !isBadgeExpired(specConfig.expiresAt)
+      specConfig &&
+      isBadgeActive(specConfig.status, specConfig.metadata?.expiresAt || null)
```

Comment:
@rahulrumalla There are  open proposals, 


```JSON
      {
        "id": "0x034ef9bb0c706c9a0c01afe6d98864ac94e5c037efb0e0027183de907d1a5a37",
        "space": {
          "id": "emilyfurlong.eth"
        },
        "state": "active"
      },
      {
        "id": "0x81eff3b728a65529290bf55b0f9d5293da218bc4f7891baa38c9079975ed2003",
        "space": {
          "id": "otterspace.mirror.xyz"
        },
        "state": "active"
      },
```
But they are not using `expiresAt` so I think it should be fine.?